DependencyInjection/DunglasActionExtension.php

```diff
@@ -51,7 +52,7 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->setParameter('dunglas_action.directories', $directories);
 
-        if (class_exists('Symfony\Component\Routing\Loader\AnnotationDirectoryLoader')) {
+        if (Kernel::VERSION_ID >= 30100 && class_exists('Symfony\Component\Routing\Loader\AnnotationDirectoryLoader')) {
```

Comment:
this check doesn't seem safe to me as ``symfony/kernel`` can be different of ``symfony/framework-bundle``.
maybe using ``method_exists(Controller::class, 'json')`` would be safer?